src/test/java/io/zeebe/containers/examples/AbstractExampleTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2019 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.containers.examples;
+
+import io.zeebe.containers.AbstractTest;
+import io.zeebe.containers.ZeebeBrokerNode;
+import java.time.Duration;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.testcontainers.containers.GenericContainer;
+
+/**
+ * Provides helper methods for the examples. If you plan on setting up clusters, and as a
+ * consequence cannot use the {@link org.testcontainers.junit.jupiter.Testcontainers} extension,
+ * it's recommended you also look into {@link AbstractTest#withCloseable(AutoCloseable)} as a way to
+ * ensure your containers are cleaned up after tests. Note that the Ryuk container will still ensure
+ * that they are cleaned up, but if you run many tests in parallel, closing your containers as soon
+ * as possible is best to avoid resource contention.
+ */
+abstract class AbstractExampleTest extends AbstractTest {
```

Comment:
I worry about the same inheritance for reuse problem as in AbstractTest. I think here it also makes sense to move this into some ExampleHelper class. The `getConfiguredClusterBroker` method seems to be able to be turned into a static member.

Reply:
I did think about this, and I think actually for examples it makes more sense to duplicate this code - it's nicer to be able to read the whole example in one go, all laid out, imho. So I'll update it to that.